MSBBarChart/Sources/MSBBarChartView.swift

```diff
@@ -105,9 +105,10 @@ extension MSBBarChartView {
     }
 
     private func showEntry(index: Int, entry: BarEntry, maxInterval: CGFloat) {
+        guard let maxBar = getMaxEntry(), let entryValue = Float(entry.textValue), let maxEntryValue = Float(maxBar.textValue) else { return }
         let barWidthSet = barWidth + space
         let xPos: CGFloat = yAxisLabelWidth + bothSideMargin + CGFloat(index) * barWidthSet
-        let yPos: CGFloat = translateHeightValueToYPosition(value: CGFloat(Int(entry.textValue)!) / CGFloat(maxYvalue))
+        let yPos: CGFloat = translateHeightValueToYPosition(value: CGFloat(entryValue / maxEntryValue))
```

Comment:
When chart data is empty, **entryValue** is 0 and **maxEntryValue** is 0
**yPos** becomes 0 / 0 which results in NaN
This crashes CALayer when **yPos** gets passed into **executeAnimation** #20